### DISCUSSION TITLE
Custom Track Stuck Threshold

### DISCUSSION THREAD

**Discussion by wt4vi** [lavalink-devs/Lavalink#679] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107622047/175790335-e2090b4e-7457-4f7b-a548-7f1b213a6e93.png)
I would like to know how to make this last longer

**Comment by topi314** [comment, marked as answer]
Hey I actually just merged support for setting this in https://github.com/freyacodes/Lavalink/pull/676
You can download the build from the ci linked in the readme

In the future please choose a describing title

**Reply by wt4vi** [reply]
I added `trackStuckThresholdMs: 100000` in `application.yml`, but this appears to not change the track stuck threshold

**Reply by topi314** [reply]
Did you update lavalink?

**Reply by wt4vi** [reply]
yes, i downloaded it from ci

**Reply by topi314** [reply]
Can you show me your start-up logs?

**Reply by wt4vi** [reply]
![image](https://user-images.githubusercontent.com/107622047/175805386-f3bcd9fe-4f42-450c-847c-cfab2c1e778f.png)
![image](https://user-images.githubusercontent.com/107622047/175805400-2a5bdc9d-867a-4918-9434-2964cecaa7a9.png)
![image](https://user-images.githubusercontent.com/107622047/175805408-78135e67-e737-490f-9916-9b30589c9c9f.png)

**Reply by topi314** [reply]
This change has not been merged to master but dev
And will be fully released with v3.5

**Reply by wt4vi** [reply]
i downloaded `Lavalink.jar` from the last release, but this doesn't appear to work tho

**Reply by topi314** [reply]
You should download the last jar from the dev branch from the ci as I said above

**Reply by wt4vi** [reply]
i found the right one - https://ci.fredboat.com/viewLog.html?buildId=9438&buildTypeId=Lavalink_Build&tab=artifacts&branch_Lavalink=refs%2Fheads%2Fdev
